fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanAccrualActivityProcessingServiceImpl.java

```diff
@@ -204,7 +204,14 @@ private boolean validateActivityTransaction(@NotNull LoanRepaymentScheduleInstal
 
     private void reverseAccrualActivityTransaction(LoanTransaction loanTransaction) {
         loanTransaction.reverse();
-        loanTransaction.updateExternalId(null);
+
```

Comment:
WE don't need this validation. Simply removing the part where we set the external id to null is sufficient!